### PR TITLE
ACMS-1498: Fixing missing client id for headless.

### PR DESCRIPTION
**Pull request by apathak18**
**Motivation**

Fixes ACMS-1498

**Proposed changes**
Adding client id for headless

### CODE REVIEW

File: modules/acquia_cms_headless/src/Service/StarterkitNextjsService.php [hunk 1]
```diff
@@ -160,9 +169,11 @@ public function createHeadlessConsumer(array $consumer_data) {
       $user = $this->getHeadlessUserData();
 
       if (!empty($user) && empty($consumers)) {
+
         $this->consumerSecret = $this->createHeadlessSecret();
         $consumer = Consumer::create();
         $consumer->set('label', $consumer_data['site-name']);
+        $consumer->set('client_id', $this->clientId);
```
Comment (rajeshreeputra) [inline modules/acquia_cms_headless/src/Service/StarterkitNextjsService.php:176]: client ID should be machine name of label, example:
Label = Default Consumer
client ID = default_consumer

Comment (apathak18) [reply modules/acquia_cms_headless/src/Service/StarterkitNextjsService.php:176]: As discussed this in DSM.. marking as resolved.

**Review by deepakmishra2 — APPROVED** [review]
LGTM

**Review by deepakmishra2 — APPROVED** [review]
Looks good

**Pull request merged**